src/index.d.ts

```diff
@@ -1487,6 +1573,30 @@ declare namespace admin.auth {
      */
     deleteUser(uid: string): Promise<void>;
 
+    /**
+     * Deletes the users specified by the given uids.
+     *
+     * Deleting a non-existing user won't generate an error. (i.e. this method
```

Comment:
Deleting a non-existing user won't generate an error (i.e. this method is idempotent).

Reply:
Done.